views/publication/summary.layout.gohtml

```diff
@@ -1,134 +1,137 @@
-                        {{.Publication.Title}}
-                    {{else}}
-                        Untitled record
-                    {{end}}
-                    {{else}}
-                    <a href="{{.PublicationURL}}">
-                    {{if .Publication.Title}}
-                        {{.Publication.Title}}
-                    {{else}}
-                        Untitled record
-                    {{end}}
-                    </a>
-                    {{end}}
-                </h4>
-
-                <div class="c-meta-list c-meta-list-horizontal">
-                    {{range .Publication.SummaryParts}}
-                    <span class="c-meta-item">{{.}}</span>
-                    {{end}}
-                </div>
+                        <h4 class="mb-0">
+                            {{if eq .UserRole "curator"}}
+                            {{if .Publication.Title}}
+                                {{.Publication.Title}}
+                            {{else}}
+                                Untitled record
+                            {{end}}
+                            {{else}}
+                            <a href="{{.PublicationURL}}">
+                                <span class="list-group-item-title">
+                                    {{if .Publication.Title}}
+                                        {{.Publication.Title}}
+                                    {{else}}
+                                        Untitled record
+                                    {{end}}
+                                </span>
+                            </a>
+                            {{end}}
+                        </h4>
+                        <ul class="c-meta-list c-meta-list-inline">
+                            {{range .Publication.SummaryParts}}
+                            <li class="c-meta-item">{{.}}</li>
+                            {{end}}
+                        </ul>
 
-                {{with .Publication.Author}}{{partial "shared/contributors" (dict "Contributors" . "ContributorsURL" ($.PublicationURL | querySet "show" "contributors"))}}{{end}}
+                        {{with .Publication.Author}}{{partial "shared/contributors" (dict "Contributors" . "ContributorsURL" ($.PublicationURL | querySet "show" "contributors"))}}{{end}}
+                    </div>
 
-                {{with .Publication.RelatedOrganizations}}{{partial "shared/related_organizations" (dict "RelatedOrganizations" . "ContributorsURL" ($.PublicationURL | querySet "show" "contributors"))}}{{end}}
+                    <div class="collapsible-card-content collapse" card-content>
+                        <div class="vstack gap-3">
+                            <div class="d-flex align-items-center">
+                                <i class="if if-building if--small if--muted me-2"></i>
+                                {{if .Publication.RelatedOrganizations}}
+                                    {{with .Publication.RelatedOrganizations}}
```

Comment:
`with` is not necessary anymore. That `if` should have solved that already.

Or you can use this construct (starting at line 99) `{{ with .Publication.RelatedOrganizations }} {{ else }} {{ end }}`

Reply:
Whenever I use `if` or mix `with` and `else` I get an internal server error (this is why I nested it in the first place). This could be a dev container issue (I can't use the project service yet locally), or something else is off.

Did you run that code, did it work?

Reply:
I have updated your code. See two latest commits

Reply:
Thank you, seems work!